apparmor.d/groups/ssh/sshd

```diff
@@ -43,14 +43,17 @@ profile sshd @{exec_path} flags=(attach_disconnected) {
 
   ptrace (read,trace) peer=unconfined,
 
+  network inet  stream,
+  network inet6 stream,
+
   @{exec_path} mrix,
 
-  /{usr/,}bin/{,b,d,rb}ash         rUx,
-  /{usr/,}bin/{c,k,tc,z}sh         rUx,
+  /{usr/,}bin/{,b,d,rb}ash         rPUx,
```

Comment:
Alright, I got you.
I haven't experienced any breakage on Debian-derivatives, since `bash` there is always personal and all other process-originating shells uses `dash`.